README.md

```diff
@@ -1,52 +1,58 @@
-# Magento 2.2.6 Magento Commerce (Cloud)
+# Magento 2.2.6 Magento Commerce Cloud
 
-This sample repository helps you deploy a Magento 2.2.6 Enterprise Edition (EE) instance in the cloud. You must be a licensed user of Magento Magento Commerce (Cloud) to use the example in this repository.
+This repository contains a sample Magento Commerce (on-premise) version 2.2.6 instance for you to deploy in the cloud. You must have an active Magento Commerce Cloud user license to use the example in this repository.
 
-This example is based on using the Composer to load dependencies and get the Magento vendor folders.
+The example requires the use of [Composer](https://getcomposer.org/doc/) to load and manage dependencies and Magento vendor folders.
 
-## Create an authorization file
-To enable you to install and update the Magento software, you must have an `auth.json` file in your project's root directory. `auth.json` contains your Magento EE [authorization credentials](http://devdocs.magento.com/guides/v2.2/install-gde/prereq/connect-auth.html).
+## Authentication
+You must have an authentication key to access the Magento Commerce repository and to enable install and update commands for your Magento Commerce Cloud project. 
+The following method is best to prevent accidental exposure of credentials, such as pushing an `auth.json` file to a public repository.
 
-In some cases, you might already have `auth.json` so check to see if it exists and has your authentication credentials before you create a new one.
+To add authentication keys using an environment variable:
 
-To create a new `auth.json` in the event you don't have one:
+1.  In the _Project Web UI_, click the configuration icon in the upper left corner.
 
-1.  Copy the provided sample using the following command:
+1.  In the _Configure Project_ view, click the **Variables** tab.
 
-        cp auth.json.sample auth.json
-2.  Open `auth.json` in a text editor.
```

Comment:
Maybe we should leave `auth.json` example dor Docker usage?

Reply:
I can do that. I can add some Docker stuff in there.